README.md

```diff
@@ -185,6 +191,75 @@ module.exports = {
 };
 ```
 
+By passing a `library` option, you can switch from building a React application to a web library. Set `library`
+to the name of your library as it would be exposed on the `window` object.
+
+_Example: create a `taskcluster` web library:_
+
+```js
+module.exports = {
+  use: [
+    ['neutrino-preset-mozilla-rpweb', {
+      library: 'taskcluster'
+    }]
+  ]
+};
+```
+
+By passing a `components` option, you can switch from building a React application to a set of React components.
+Set `components` to `true`.
+
+_Example: create a `taskcluster` React components library:_
+
+```js
+module.exports = {
+  use: [
+    ['neutrino-preset-mozilla-rpweb', {
+      components: true
+    }]
+  ]
+};
+```
+
+## Building libraries and components
+
+When building libraries and components, they are output to the `lib` directory by default when running `neutrino build`.
+Using the quick start example above as a reference:
```

Comment:
Correct me if I'm wrong, but the user will need to develop his components under `src/components`. Is it mentioned anywhere?